Tests/Parser/Client/fixtures/library.yml

```diff
@@ -528,3 +527,33 @@
     type: library
     name: Axios
     version: "1.2.0"
+-
+  user_agent: HTMLParser/1.6
+  client:
+    type: library
+    name: HTML Parser
+    version: "1.6"
+-
+  user_agent: python-urllib3/1.26.9
+  client:
+    type: library
+    name: Python urllib
+    version: 1.26.9
+-
+  user_agent: msray-plus
+  client:
+    type: library
+    name: msray
+    version: ""
+-
+  user_agent: Slim Framework
+  client:
+    type: library
+    name: Slim
+    version: ""
+-
+  user_agent: Fuzz Faster U Fool v1.5.0-dev
+  client:
+    type: library
+    name: Fuzz Faster U Fool
```

Comment:
FFUF seems to be used way more often, and since official logo uses abbreviation, I wonder if the name should be FFUF, too